packages/pwa-kit-runtime/src/ssr/server/test_fixtures/extensions/extension-with-bad-setup-server/setup-server.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+/* istanbul ignore next */
+const extension = ({app}) => {
+    throw new Error('extension will fail to initialize because of me')
+}
+
+module.exports = {
+    default: extension
+}
```

Comment:
Why is there duplication of all these files?

Reply:
because `pwa-kit-dev` tests import es6/typescript files but `pwa-kit-runtime` tests import commonjs files.